From 6ef6114a5d733e35b699f58ad83f8dfd67c9a09e Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Fri, 29 Jul 2022 08:53:36 +0300
Subject: [PATCH] [SW-0] gaudi3: add d2d efuse dword to sec data

Add efuse offset 0x3B to sec data structure. This is to extract
the d2d en/dis bit in efuse and store in boot info data.

Change-Id: Iab649027a34fa500c80d9ff670168439c8f296e2
---

diff --git a/arch/gaudi3/arc/security.c b/arch/gaudi3/arc/security.c
index 24b7103..8c47a6b 100644
--- a/arch/gaudi3/arc/security.c
+++ b/arch/gaudi3/arc/security.c
@@ -640,6 +640,9 @@
 {
 	arcboot_rom_global.sec_data.efuse_sys_cfg =
 			arcboot_rom_global.config_fuse;
+
+	arcboot_rom_global.sec_data.efuse_sys_cfg2 =
+			arcboot_rom_global.config2_fuse;
 }
 
 void arcboot_rom_security_init(void)
